test/data/Cardano/Wallet/Api/WalletPutData.json

```diff
@@ -0,0 +1,11 @@
+{
+    "seed": -3282480895854715413,
+    "samples": [
+        {},
+        {},
+        {},
+        {
+            "name": "ro8n&(V$]%dJ吚 (v]QSu1𩈾DfZ|*w?\\Iy,X%d9k}[𦴽YFJtW?7ZJv_-jYz_KpB`iE@$*4aV)`𠪚V`Q&0O7=ZmB<KG3Y5$\\>WO$1(/4子tmkNd;Zcte𥗼fay>nqh𨃮wQwW~u:`,pzhSB6|&~p}\"dS6🢓_3#*+f#5𠷓>:-Yq58Q_:𠷒as0/^j7;^BafT3I3y䖜jY0<CiwxqJ.Ofx<\" 1FDU9HrlD$7𨸴cEIq+*>~"
```

Comment:
That's the kind of name I imagine @piotr-iohk would use on a wallet yeah :no_mouth:

Reply:
Buried deep within this name is the character "䖜", which apparently is "the fighting sound made by two tigers" (https://en.wiktionary.org/wiki/%E4%96%9C)